src/i18n/locales/en-US.ts

```diff
@@ -1155,6 +1155,9 @@ export default {
         },
     },
     insights: {
+        insightsUpdateTitle: 'New Insights Dashboard',
+        insightsUpdateContent:
+            'Check out the new Insights dashboard to see your team’s steps of faith, interactions and challenges. Now you can see and celebrate what God is doing in your movement.',
```

Comment:
```suggestion
            'Check out the new Insights dashboard to see your team’s steps of faith, interactions and challenges. Now you can see and celebrate what God is doing in your community.',
```

Although I guess web still calls them `ministries`... Idk